Sebastian support

jakefentress opened the pull request:
This PR adds Sebastian support by creating a findSebs helper to run during message parsing which will find each :seb: reference and replace it with the applicable image element.

A video walkthrough: https://share.cleanshot.com/NZD5f06Y

bedrich commented on client/js/helpers/parse.ts:
```diff
@@ -212,6 +215,25 @@ function parse(text: string, message?: ClientMessage, network?: ClientNetwork) {
+			const name = textPart.seb[1];
+			const filename: string = textPart.seb[0];
+			const metadata = filename.match(/[a-z0-9]+_?(\d+)?/i);
+			const height = metadata[1] !== "undefined" ? metadata[1] : "auto";
```
I'm not sure if the array returned by `match` coerces the literal value of `undefined` to be a string (i.e. `"undefined"`), but if that's not the case, this conditional will *always* return `metadata[1]` unless we had a seb filename with a pattern like `<name>_undefined.<ext>`. 

I think we have two options here:

```ts
const height = metadata[1] !== undefined ? metadata[1] : "auto";
```

or

```ts
const height = metadata[1] || "auto";
```

bedrich reviewed (changes requested):
I'm not sure if you made a note about this on the video because of the lack of audio. If there is a bit of context I'm missing here, feel free to ignore it and merge.

jakefentress commented:
@bedrich I've added in the updates to 'height' and also moved values over to the constants file, changing the sebs array over to the format of the object we auto-generate at https://d2nj10p1euioo5.cloudfront.net/list.json. Ultimately, we can look into fetching that remove file so we don't need to update this code for new sebs, but that's a whole thing to sort out and that file has a couple of errors to fix.

bedrich reviewed (approved):
🚀
